code/builders/builder-manager/package.json

```diff
@@ -52,7 +52,7 @@
     "@yarnpkg/esbuild-plugin-pnp": "^3.0.0-rc.10",
     "browser-assert": "^1.2.1",
     "ejs": "^3.1.8",
-    "esbuild": "^0.18.0",
```

Comment:
@valentinpalkovic do you remember why we are on 0.18? I reckon something happened in the past but I just don't remember

Reply:
whats the best way to prove everything is fine other than CI being happy?

if you can point me in the right direction, i can try do further testing

Reply:
Hmm,.. maybe this:
https://github.com/storybookjs/storybook/issues/26112